### DISCUSSION TITLE
#map list variable <name> no longer exports a variable

### DISCUSSION THREAD

**Discussion by miguelalbuquerque** [scandum/tintin#100] [Bugs]
Greetings!

It seems the command #map list variable <name> no longer exports a variable.

Command: #map list {variable} {maplist}
Command: #var maplist
Result: #VARIABLE {maplist} {} 
Command: #show &maplist[]
Result: 0

Worked (at least) in version 2.02.4
No longer works in 2.02.20.

Unless the command usage has changed? :)
Any help is appreciated!

Thank you.

**Comment by scandum** [comment]
The following should work:
```
#map list {%*} {variable} {maplist}
```

I broke the old syntax trying to fix a bug, it's on my todo list to fix.

**Reply by miguelalbuquerque** [reply]
Wonderful.
Thanks a lot for the feedback!
